nimporter.py

```diff
@@ -924,11 +924,22 @@ def _find_extensions(cls, path, exclude_dirs=set()):
             A list of Path objects. File Paths indicate a Nim Module. Folder
             Paths indicate Nim Libraries.
         """
-        exclude_dirs = {p.expanduser().absolute() for i in exclude_dirs}
+
+        exclude_dirs = [ os.path.abspath(os.path.expanduser(p)) for p in exclude_dirs ]
```

Comment:
In order to be consistent with the code style present in the rest of `nimporter.py`, can you remove the preceding and trailing whitespace for `exclude_dirs`? For example: `[foo]` instead of `[ foo ]`.

Also, can you use the `pathlib.Path` functionality instead of the older `os.path.<etc>` functions to be more consistent with the rest of the path handling functions in the file?